### PR TITLE
docs: improve code sections by removing noise

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/components/notification/notification.stories.ts [hunk 1]
```diff
@@ -235,9 +235,7 @@ export const MultipleNotifications: StoryObj = {
 const meta: Meta = {
   decorators: [
     (story, context) =>
-      html`<div
-        style="padding: 2rem;display: flex;gap: var(--sbb-spacing-fixed-4x);flex-direction: column;"
-      >
+      html`<div style="display: flex;gap: var(--sbb-spacing-fixed-4x);flex-direction: column;">
```
Comment (DavideMininni-Fincons) [inline src/components/notification/notification.stories.ts:238]:
```suggestion
      html`<div style="display: flex; gap: var(--sbb-spacing-fixed-4x); flex-direction: column;">
```

File: src/components/toast/toast.stories.ts [hunk 2]
```diff
@@ -192,10 +189,8 @@ const meta: Meta = {
         SbbToastElement.events.didClose,
       ],
     },
-    backgrounds: {
-      disable: true,
-    },
     docs: {
+      story: { inline: false, iframeHeight: '200px' },
```
Comment (DavideMininni-Fincons) [inline src/components/toast/toast.stories.ts:193]: Curiosity: why this line?

**Review by DavideMininni-Fincons — COMMENTED** [review]
Great improvement!

File: src/components/toast/toast.stories.ts (hunk 2 above)
Comment (jeripeierSBB) [reply src/components/toast/toast.stories.ts:193]: added comment

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**